apps/server/test/modules/authentication/account.e2e-spec.ts

```diff
@@ -163,4 +170,164 @@ describe('Account Controller (e2e)', () => {
 				.expect(400);
 		});
 	});
+
+	describe('[GET]', () => {
```

Comment:
The more automated tests the better.
But you only need e2e tests for stuff you can't check with unit tests. Some tests are not really necessary, because they are already covered, e.g. tests that check authorization.

Reply:
I have documented these response types in Swagger. I think things that are documented should also be tested.